auth_signup_verify_email/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,6 @@
+To configure this module, you need to:
+
+* `Properly configure your outgoing email server(s)
+  <https://www.odoo.com/forum/help-1/question/how-to-configure-email-gateway-282#answer_290>`_.
+* Go to *Settings > General Settings -> General settings*, search for
```

Comment:
```suggestion
* Go to *Settings > General Settings*, search for
```

Reply:
I think that is correct:
![general_settings](https://user-images.githubusercontent.com/731270/58971221-418e7500-87bb-11e9-9b1b-c3d4cf3e7864.png)

Reply:
Last changes are made by Yajo... it's correct squash?

Reply:
For me it's OK, but thanks for asking.